worlds/LauncherComponents.py

```diff
@@ -131,8 +131,9 @@ def _install_apworld(apworld_src: str = "") -> Optional[Tuple[pathlib.Path, path
             found_already_loaded = True
             break
     if found_already_loaded:
-        raise Exception(f"Installed APWorld successfully, but '{module_name}' is already loaded,\n"
-                        "so a Launcher restart is required to use the new installation.")
+        raise Exception(f"Installed APWorld successfully, but '{module_name}' is already installed,\n"
```

Comment:
I don't like this change from "loaded" to "installed".

After https://github.com/ArchipelagoMW/Archipelago/pull/3226, it could be installed and not require a restart because it's not loaded.
So what matters is that it's loaded, not that it's installed.

Reply:
After #3226, the message will need to be changed to omit the restart bit anyway.

In my eyes, if I saw just that message without having the launcher open, and it said loaded, I'd be a little confused since I don't have anything "loaded" atm.
If I saw that message with the launcher open, and it said installed, I would understand what it meant.

Reply:
> After #3226, the message will need to be changed to omit the restart bit anyway.

Why? That doesn't provide a way to unload worlds, so it would still need to be restarted if the world is loaded.

Reply:
Ah, my mistake, I didn't read through that pr thoroughly enough.

The rest of the response still stands, though.

Reply:
Supporting the user's incorrect ideas is not a good way to avoid confusion.

If the user is thinking they don't have anything loaded when they see this, that idea is just wrong.

Reply:
I'm going to change it to your suggestion. I don't agree with it still, but it's way too much time spent on a single word.